### PR TITLE
refactor(triggered): make "Open project" a plugin

### PR DESCRIPTION
**Pull request by c-dinkel**

### CODE REVIEW

File: public/js/plugins.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
 export const officialPlugins = [
+  {
+    name: 'Open file',
+    src: '/src/triggered/OpenFile.js',
+    icon: 'coronavirus',
```
Comment (dlabordus) [inline public/js/plugins.js:5]: Is this correct, "coronavirus"? Or some typing in the wrong editor?

Comment (c-dinkel) [reply public/js/plugins.js]: Sorry, that was a placeholder icon that we never updated.

**Comment by c-dinkel** [comment]
This is ready to be merged without any unit tests, since the new plugin consists entirely in completely untestable Browser API code.

https://github.com/com-pas/compas-open-scd/tree/open-compas-plugin is waiting for this.

File: src/open-scd.ts [hunk 2]
```diff
@@ -193,7 +171,6 @@ export class OpenSCD extends Setting(
     if (ctrlAnd('z')) this.undo();
     if (ctrlAnd('l')) this.logUI.open ? this.logUI.close() : this.logUI.show();
     if (ctrlAnd('m')) this.menuUI.open = !this.menuUI.open;
-    if (ctrlAnd('o')) this.fileUI.click();
```
Comment (dlabordus) [inline src/open-scd.ts:196]: Christian, maybe discus if this is a problem. Users are now not able to open a project anymore using ctrl+o.

**Review by dlabordus — COMMENTED** [review]
See my comment on the ctrl+o.

File: src/open-scd.ts (hunk 2 above)
Comment (c-dinkel) [reply src/open-scd.ts:196]: I added a shortcut that opens the first `loader` plugin on `ctrl+o`

**Review by JakobVogelsang — APPROVED** [review]
LGTM

File: public/js/plugins.js [hunk 3]
```diff
@@ -21,11 +21,18 @@ export const officialPlugins = [
...
+    icon: 'folder_open',
+    default: true,
+    kind: 'loader'
+  },
+  {
+    name: "Import IEDs",
+    src: "/src/triggered/ImportIEDs.js",
+    icon: "snippet_folder",
+    default: true,
+    kind: "triggered"
```
[the remark this answers is not in the document]
Comment (c-dinkel) [reply public/js/plugins.js:35]: Loader is currently the type of plugin displayed at the very top of the menu and clickable without a `doc` loaded. Import IEDs should only be available when a doc is already loaded and could just stay at the top of the `triggered` plugin section.

**Pull request merged**